index.html

```diff
@@ -949,10 +957,20 @@ <h3>
         <ol>
```

Comment:
In the algorithms for "when a device becomes available/unavailable on the system" we currently use "|device|'s [=relevant global object=]'s {{Navigator}} object" (window.navigator) which won't exist in a service worker. Instead we need to check the relevant global object to decide whether to use Navigator.hid or WorkerNavigator.hid.

Reply:
Done.